newIDE/app/src/EventsSheet/InstructionEditor/InstructionTypeSelector.js

```diff
@@ -215,6 +215,11 @@ export class InstructionTypeSelector extends Component {
     });
   };
 
+  //// autofocus on search
+  componentDidMount() {
+    this._searchBar.focus();
+    };
```

Comment:
There are two extra spaces before `}`. ;)

You can run `npm run format` (or `yarn format`) in the app folder to automatically format all files.
Note that this will make changes to a lot of files because it has been a long time since I've not run it - I should find a way to enforce this `npm run format` command :)

Reply:
I will run it and do another commit if you would like :)
I just didn't want to make edits to a lot of files to get this in
The enforce sounds like a good idea!